src/util/cloud_utils.js

```diff
@@ -35,7 +34,7 @@ function get_signed_url(params) {
         s3DisableBodySigning: disable_s3_compatible_bodysigning(params.endpoint),
         region: 'eu-central-1',
         httpOptions: {
-            agent: agent
+            agent: http_utils.get_unsecured_agent(params.endpoint)
```

Comment:
I am pretty sure that this is unneeded because we only call getSignedUrl() on this s3 client and the actual http call will be made by the client that gets this url (endpoint or browser). No harm by passing it, but maybe just worth noting that in a comment so that it will be clearer that this is not really a must.

Reply:
It does not harm, would you like me to remove that agent?

Reply:
no. just a small comment would be nice because as a reader it's hard to understand if this was intentional or not.

Reply:
Comment added